src/clients/servicebus/ServiceBusClient.js

```diff
@@ -19,6 +17,13 @@ function sendStringMessage(queue, message) {
       return reject('No message to be sent to service bus.');
     }
 
+    let client;
```

Comment:
Please raise an app insight error event in addition to rejecting the `Promise` https://github.com/Microsoft/ApplicationInsights-node.js#track-custom-telemetry